packages/server/README.md

```diff
@@ -0,0 +1,45 @@
+# Court Backend server
+
+This server aims to provide different kind of services to complement the logic implemented at the smart contracts level.
+
+## Setup
+
+To run the server simply create your own `.env` file, feel free to follow the provided `.env.sample` template.
+Once you have done that, make sure you have postgres installed and running and simply run the following commands:
+
+```bash
+npm i
+npx sequelize db:create
+npx sequelize db:migrate
+npm start
+```
+
+Remember to append `NODE_ENV=production` if needed.
+
+## Endpoints
+
+All the provided endpoints are `Content-Type: application/json`
+
+### 1. Subscriptions
+
+#### 1.1. Create
+
+- URL: /subscriptions
+- Method: POST
+- Body: 
+  - email: Email address, string mandatory
+  - address: Ethereum address, string not mandatory
+  - amount: Amount of ANJ converted, string not mandatory
```

Comment:
Not sure if this is really necessary at this point; especially since it's going to get outdated as soon as the user transfers ANJ.

Reply:
Yea, let's drop it 👍